app/assets/stylesheets/season.css.scss

```diff
@@ -0,0 +1,7 @@
+body.season-bg {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Properties should be ordered -moz-background-size, -o-background-size, -webkit-background-size, background-size, color